frontend/src/modules/lobby/view/Lobby.vue

```diff
@@ -55,9 +58,8 @@ export default {
       this.connectionState = "CONNECTED";
```

Comment:
do we even use this connectionState

Reply:
Uhh nope I'll remove it next time i clean up